### PR TITLE
Change quote command to use quotes from characters

### PR DESCRIPTION
**Pull request by duncte123**

### CODE REVIEW

File: utils/utils.py [hunk 1]
```diff
@@ -124,15 +126,40 @@ def render_dialog(text: str, character: str, background: str = 'camp') -> BytesI
...
+        'goro': 0x39fe19,
+        'yoshinori': 0xc07c35,
+        'yuri': 0xff73bb,
+        'seto': 0x9dd5e6,
+        'felix': 0xbdd7e6,
+        'eduard': 0xd37fb7,
+        'lee': 0x00aa6c,
+        'kieran': 0xe7ab0d,
+        'naoto': 0x227abe,
+    }
```
Comment (zsotroav) [inline utils/utils.py:162]: `assets/chars.json` has every color in hex, no need to store it multiple times. Use the JSON, or port the JSON to a Python-compatible format as we did with #14 Fix/Convert's dictionary.

**Review by zsotroav — CHANGES_REQUESTED** [review]

File: utils/utils.py (hunk 1 above)
Comment (duncte123) [reply utils/utils.py:162]: Thanks for the tip

**Review by zsotroav — CHANGES_REQUESTED** [review]
The code looks good, and I'm just nitpicking.

**Review by zsotroav — APPROVED** [review]

File: cogs/fun.py [hunk 2]
```diff
@@ -38,12 +41,54 @@ def calculate_score(first: discord.Member, second: discord.Member, ship_messages
...
+
+        if character:
+            char = character.lower()
+            if char not in self.charNames:
+                await ctx.send(f'I don\'t know who "{character}" is')
+                return
+        else:
+            char = random.choice(self.charNames)
+
+        with open(f'./assets/quote/{char}.txt') as lines:
```
Comment (iamdeja) [inline cogs/fun.py:78]: Doesn't this have to be closed somewhere

**Review by iamdeja — APPROVED** [review]

**Pull request merged**